stroyprombeton/tests/tests_views.py

```diff
@@ -1079,14 +1055,111 @@ def test_product_images(self):
         product = models.Product.objects.get(id=PRODUCT_WITH_IMAGE)
         series = models.Series.objects.first()
         product.options.update(series=series)
-        response = self.client.get(series.url)
+        response = self.client.get(series.url)  # Ignore CPDBear
         image = response.context['product_images'][110]
         self.assertTrue(image)
         self.assertTrue(image.image.url)
 
     def test_product_image_button(self):
         """Series page should contain button to open existing product image."""
         # product with image
+        product = models.Product.objects.get(id=PRODUCT_WITH_IMAGE)  # Ignore CPDBear
+        response = self.client.get(product.category.url)
+        self.assertContains(response, 'table-photo-ico')
+
+
+@tag('fast', 'catalog')
+class SectionFirst(BaseCatalogTestCase):
+    fixtures = ['dump.json']
+
+    def test_200_ok(self):
+        section = models.Section.objects.create(
+            name='First section',
+            slug='fist',
+            page=ModelPage.objects.create(
+                name='First section',
+            ),
+        )
+        models.Product.objects.active().update(section=section)
+        assert section.page
+        response = self.client.get(section.url)
+        self.assertEqual(200, response.status_code)
+
+
+# @todo #669:60m  Create tests for Section.
+#  Use Section tests draft below. Rm current SectionFirst draft.
+@unittest.skip
+@tag('fast', 'catalog')
+class Section(BaseCatalogTestCase):
+    fixtures = ['dump.json']
+
+    def setUp(self):
+        self.section = models.Section.objects.filter(options__isnull=False).first()
+
+    def get_section_url(
+        self,
+        section: models.Section = None,
+    ):
+        section = section or self.section
+        return reverse('section', kwargs={'section_slug': section.slug})
+
+    def get_section_page(self, *args, **kwargs):
+        return self.client.get(self.get_section_url(*args, **kwargs))  # Ignore CPDBear
+
+    def get_section_soup(self, *args, **kwargs) -> BeautifulSoup:
+        section_page = self.get_section_page(*args, **kwargs)
+        return BeautifulSoup(
+            section_page.content.decode('utf-8'),
+            'html.parser'
+        )
+
+    def test_options_are_from_section(self):
+        response = self.client.get(self.get_section_url(self.section))
+        self.assertTrue(
+            all(option.section == self.section for option in response.context['products'])
```

Comment:
```suggestion
            for option in response.context['products']:
                self.assertEqual(option.section, self.section)
```

I'd prefer this approach because it will show the diff in case of failed test. Up to you